Glideslope.json

```diff
@@ -20088,225 +20088,173 @@
+    "latitude": 42.00283873, 
     "identifier": "ISAJ"
   }, 
   {
     "receptionRange": 10, 
-    "bearing": 90.984, 
+    "bearing": 89.99, 
     "airportICAO": "KORD", 
-    "elevation": 659, 
+    "elevation": 679, 
     "name": "GS", 
-    "longitude": -87.914257, 
-    "associatedRunwayNumber": "09R", 
+    "longitude": -87.91840744, 
+    "associatedRunwayNumber": "9R", 
     "frequency": 110500, 
-    "glideslope": 3.00089992, 
-    "latitude": 41.985588, 
+    "glideslope": 3.0008999, 
+    "latitude": 41.98389856, 
     "identifier": "IJAV"
   }, 
   {
     "receptionRange": 10, 
-    "bearing": 90.76, 
+    "bearing": 89.88, 
     "airportICAO": "KORD", 
-    "elevation": 672, 
+    "elevation": 679, 
     "name": "GS", 
-    "longitude": -87.92741965, 
-    "associatedRunwayNumber": "10", 
+    "longitude": -87.93161491, 
+    "associatedRunwayNumber": "10L", 
     "frequency": 111100, 
     "glideslope": 3.0008988, 
-    "latitude": 41.96817829, 
+    "latitude": 41.96899654, 
     "identifier": "IMED"
   }, 
   {
     "receptionRange": 10, 
     "bearing": 89.87, 
     "airportICAO": "KORD", 
-    "elevation": 672, 
+    "elevation": 679, 
     "name": "GS", 
-    "longitude": -87.931523, 
+    "longitude": -87.93159598, 
     "associatedRunwayNumber": "10C", 
     "frequency": 108950, 
-    "glideslope": 3.00140094, 
-    "latitude": 41.9657, 
+    "glideslope": 3.0008987, 
+    "latitude": 41.96570178, 
     "identifier": "ISXH"
   }, 
   {
     "receptionRange": 10, 
-    "bearing": 89.953, 
+    "bearing": 89.87, 
     "airportICAO": "KORD", 
-    "elevation": 676, 
+    "elevation": 679, 
     "name": "GS", 
-    "longitude": -87.927893, 
+    "longitude": -87.92791307, 
     "associatedRunwayNumber": "10R", 
-    "frequency": 110900, 
-    "glideslope": 3.00140094, 
-    "latitude": 41.957217, 
-    "identifier": "IBYW"
-  }, 
-  {
-    "receptionRange": 10, 
-    "bearing": 140.208, 
-    "airportICAO": "KORD", 
-    "elevation": 645, 
-    "name": "GS", 
-    "longitude": -87.911639, 
-    "associatedRunwayNumber": "14L", 
-    "frequency": 110900, 
-    "glideslope": 3.00140104, 
-    "latitude": 42.000825, 
-    "identifier": "IOHA"
-  }, 
-  {
-    "receptionRange": 10, 
-    "bearing": 140.19, 
-    "airportICAO": "KORD", 
-    "elevation": 668, 
-    "name": "GS", 
-    "longitude": -87.932006, 
-    "associatedRunwayNumber": "14R", 
-    "frequency": 109750, 
-    "glideslope": 3.00140095, 
-    "latitude": 41.987725, 
-    "identifier": "IORD"
-  }, 
+    "frequency": 110750, 
+    "glideslope": 3.0008987, 
+    "latitude": 41.95720417, 
+    "identifier": "IIZJ"
+  },
   {
     "receptionRange": 10, 
-    "bearing": 221.802, 
+    "bearing": 221.41, 
     "airportICAO": "KORD", 
-    "elevation": 646, 
+    "elevation": 679, 
     "name": "GS", 
-    "longitude": -87.881336, 
+    "longitude": -87.87973646, 
     "associatedRunwayNumber": "22L", 
     "frequency": 110100, 
     "glideslope": 3.00221401, 
```

Comment:
The last 5 digits do not match with the "bearing" above.

Reply:
I fixed that.